state/remote/atlas.go

```diff
@@ -48,7 +48,7 @@ func atlasFactory(conf map[string]string) (Client, error) {
 
 	parts := strings.Split(name, "/")
 	if len(parts) != 2 {
-		return nil, fmt.Errorf("malformed name '%s'", name)
+		return nil, fmt.Errorf("malformed name '%s' (valid format is 'username/name')", name)
```

Comment:
Makes sense - maybe changing the docs is enough then...
I was following the Atlas Getting Started Guide and couldn't figure out what was wrong - had to look thru source code to find the error message and see what was failing.